### ISSUE TITLE
Clean up obsolete simulated satellite product generation from UPP

### ISSUE THREAD

**Issue by WenMeng-NOAA** [NOAA-EMC/UPP#690]
The following satellite/instruments might not be in operational now:

- ssmi_f13-20
- imgr_mt2
- imgr_mtlr
- imgr_g11
- imgr_g12
- seviri_m10
- imgr_insat3d
- imgr_g15
- imgr_g13

There are no UPP ID definitions for above simulated satellite products in post_avblflds.xml though UPP IDs are reserved in source code for output. (e.g. [ssmi_f13](https://github.com/NOAA-EMC/UPP/blob/develop/sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f#L1763))
Also there are no grib2 names assigned for above simulated satellite products at
https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-3-192.shtml
I would think they were added in UPP starting from grib1 output. Can we clean up them from UPP and release UPP ID numbers for new varaibles? @fossell @hertneky @HuiyaChuang-NOAA

**Comment by hertneky** [comment]
@WenMeng-NOAA You're right, most of those never had Grib2 capabilities. The only ones in that list that did were the GOES 11/12 and it appears they are still in the grib2 table (e.g. SBT112/SBT122...) and post_avlblflds, but as you said not in operation anymore. I think it is safe to remove them all.

**Comment by WenMeng-NOAA** [comment]
@hertneky Thanks for confirming. I will add this in our UPP to-do list.

**Comment by gspetro-NOAA** [comment]
@WenMeng-NOAA @BenjaminBlake-NOAA @ChristopherHill-NOAA Do you know if this issue has been handled on the UPP code side? On the documentation side, the tables have been updated, so as long as the changes are reflected in the code, it could be closed. I don't see many of these in `post_avblflds.xml`, but there seem to be stray mentions in some code routines. Should I open a PR to remove those?

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA The updates in UPP source code haven't been made yet.

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA The updates in  https://github.com/NOAA-EMC/UPP/blob/develop/sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f are needed to clean up those satellite configurations which are no longer in service.

**Comment by gspetro-NOAA** [comment]
Ok, I will see if I can work on this in the next couple weeks. Seems fairly straightforward.

**Comment by gspetro-NOAA** [comment]
@WenMeng-NOAA Is there a difference between ssmi_f16-20 and ssmisf16-20? The code seems to mention both, and I'm not sure what the status on ssmis is.

**Comment by WenMeng-NOAA** [comment]
> [@WenMeng-NOAA](https://github.com/WenMeng-NOAA) Is there a difference between ssmi_f16-20 and ssmisf16-20? The code seems to mention both, and I'm not sure what the status on ssmis is.

@gspetro-NOAA I have no knowledge about that.

**Comment by ChristopherHill-NOAA** [comment]
@gspetro-NOAA  This mostly outdated [link](https://nsidc.org/sites/default/files/documents/technical-reference/smmr-ssmi-ssmis-sensors.pdf) from NSIDC at least provides a good descriptive summary of the sensor types (SSM/I, SSMIS).  Otherwise, the WMO OSCAR pages for [SSM/I](https://space.oscar.wmo.int/instruments/view/ssm_i) and [SSMIS](https://space.oscar.wmo.int/instruments/view/ssmis) show the current availability status for these instruments across multiple satellite platforms.

According to WMO OSCAR, the [status of the satellite instruments](https://space.oscar.wmo.int/satellitestatuses/status) originally listed at the top of this issue appears to be as follows:

```
ssmi_f13-20    (DMSP F13-F20,  SSM/I or SSMIS):
  F13, F14, F15, F16, F19, F20 are inactive;  F17 and F18 are ACTIVE
imgr_mt2       (MTSAT-2 or Himawari-7 imager) - inactive
imgr_mt1r      (MTSAT-1R or Himawari-6 imager) - inactive
imgr_g11       (GOES-11 imager) - inactive
imgr_g12       (GOES-12 imager) - inactive
seviri_m10     (METEOSAT-10 SEVIRI) - ACTIVE
imgr_insat3d   (INSAT-3D imager)  - standby mode
imgr_g15       (GOES-15 imager) - inactive
imgr_g13       (GOES-13 imager) - inactive
```
edit:  F17 and F18 are ACTIVE ~~until 07/31/2025~~

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA Let's start with the first round:
amsre: ID 483, 484, 485, 486
tim: ID 488, 489, 490, 491
You might update both post_avblflds.xml and CALRAD_WCLOUD_newcrtm.f, while keeping post_avblflds_raphrrr.xml unchanged since it is a legacy file.

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA Testing with the first round changes in your PR looks good to me. Could we incorporate the second round?
ssmi TB: ID 492 - 495
ssmis TB: ID 496 - 499
The updates should be in both post_avblflds.xml and CALRAD_WCLOUD_newcrtm.f (There are no corresponding sensor and grib2 writing configurations, so you might directly remove IDs)

**Comment by gspetro-NOAA** [comment]
> [@gspetro-NOAA](https://github.com/gspetro-NOAA) Testing with the first round changes in your PR looks good to me. Could we incorporate the second round? ssmi TB: ID 492 - 495 ssmis TB: ID 496 - 499 The updates should be in both post_avblflds.xml and CALRAD_WCLOUD_newcrtm.f (There are no corresponding sensor and grib2 writing configurations, so you might directly remove IDs)

@WenMeng-NOAA I removed IDs 492-499 from post_avblflds.xml and CALRAD_WCLOUD_newcrtm.f. I am wondering if I should also be removing anything that includes code blocks starting with `isis=='ssmi*' or if that should all stay in for the latest batch? It sounds like I just needed to remove the IGET sections, but let me know in the PR if I missed anything related to IDs 492-499.

**Comment by WenMeng-NOAA** [comment]
> > [@gspetro-NOAA](https://github.com/gspetro-NOAA) Testing with the first round changes in your PR looks good to me. Could we incorporate the second round? ssmi TB: ID 492 - 495 ssmis TB: ID 496 - 499 The updates should be in both post_avblflds.xml and CALRAD_WCLOUD_newcrtm.f (There are no corresponding sensor and grib2 writing configurations, so you might directly remove IDs)
> 
> [@WenMeng-NOAA](https://github.com/WenMeng-NOAA) I removed IDs 492-499 from post_avblflds.xml and CALRAD_WCLOUD_newcrtm.f. I am wondering if I should also be removing anything that includes code blocks starting with `isis=='ssmi*' or if that should all stay in for the latest batch? It sounds like I just needed to remove the IGET sections, but let me know in the PR if I missed anything related to IDs 492-499.

@gspetro-NOAA More coordination and evaluation are needed for that part.  Let's leave it for our future plan.
